users/vieting/experiments/switchboard/transducer/feat/experiments.py

```diff
@@ -277,6 +283,182 @@ def run_rasr_gt_stage1():
+    returnn_datasets_hash_break = get_returnn_datasets_transducer_viterbi(
+        context_window={"classes": 1, "data": 121},
+        keep_hashes=False,
+    )
+    returnn_datasets_align_ctc = get_returnn_datasets_transducer_viterbi(
+        alignment=ctc_alignment,
+        features="waveform_pcm",
+    )
+    returnn_args = {
+        "batch_size": 15000,
+        "datasets": returnn_datasets,
+        "extra_args": {
+            # data sequence is longer by factor 4 because of subsampling and 80 because of feature extraction vs.
+            # raw waveform
+            "chunking": ({"classes": 64, "data": 64 * 4 * 80}, {"classes": 32, "data": 32 * 4 * 80}),
+            "gradient_clip": 20.0,
+            "learning_rate_control_error_measure": "sum_dev_score",
+            "min_learning_rate": 1e-6,
+        },
+        "specaug_old": {"max_feature": 15},
+    }
+    returnn_args_ctc_align = copy.deepcopy(returnn_args)
+    returnn_args_ctc_align["datasets"] = returnn_datasets_align_ctc
+    returnn_args_ctc_align["extra_args"]["extern_data"] = {
+        "data": {"dim": 1, "dtype": "int16"},
+        "classes": {"dim": 88, "dtype": "int8", "sparse": True},
+    }
+    returnn_args_ctc_align["extra_args"]["min_chunk_size"] = {"classes": 2, "data": 644}
+    # Data sequence is longer by factor 4 because of subsampling and 80 because of feature extraction vs.
+    # raw waveform. Also, there are frame size - frame shift more samples at the end. This should be more correct than
+    # the version above.
+    returnn_args_ctc_align["extra_args"]["chunking"] = (
+        {"classes": 64, "data": 64 * 4 * 80 + 200 - 80},
+        {"classes": 32, "data": 32 * 4 * 80},
+    )
+    feature_args = {
+        "class": "ScfNetwork",
+        "size_tf": 256 // 2,
+        "stride_tf": 10 // 2,
+        "preemphasis": 0.97,
+        "wave_norm": True,
+        "wave_cast": True,
+    }
+    common_args = {
+        "feature_args": feature_args,
+        "lr_args": {"dynamic_learning_rate": dynamic_learning_rate_scf},
+    }
+
+    nn_args, report_args_collection = get_nn_args_baseline(
+        nn_base_args={
+            "bs15k_v1_align-ctc-conf-e400": dict(
+                returnn_args=returnn_args_ctc_align,
+                report_args={"alignment": "ctc-conf-e401"},
+                lr_args={"dynamic_learning_rate": dynamic_learning_rate_scf},
+                feature_args=feature_args,
+            ),
+        },
+        num_epochs=300,
+        evaluation_epochs=[270, 280, 290, 300],
+        prefix="viterbi_scf80_",
+    )
+    config = copy.deepcopy(nn_args.returnn_recognition_configs["viterbi_scf80_bs15k_v1_align-ctc-conf-e400"].config)
+    config["extern_data"]["data"]["dtype"] = "float32"
+    config["extern_data"]["classes"]["dtype"] = "int32"
+    nn_args.returnn_recognition_configs["viterbi_scf80_bs15k_v1_align-ctc-conf-e400"].config = config
```

Comment:
It would be nice if we could fix that in the initial returnn recognition config creation.

Reply:
Just for scf stage 1 or for the log melt stages as well?

Reply:
Both, this is not specific to any of the features.

Reply:
But if this is blocking the PR, we can also do that separately.

Reply:
Everything else is done. I would do this separately or maybe with the next PR for stage 2